spine-lua/PathConstraint.lua

```diff
@@ -345,9 +345,9 @@ function PathConstraint:computeWorldPositions (path, spacesCount, tangents, perc
 	local ddfy = 0
 	local dfx = 0
 	local dfy = 0
-	i = 0
+	local j = 0

```

Comment:
I would like to keep this `i` as it matches the reference implementation better.

Reply:
That's cool. I think I thought there was a conflict with another "local i = 0" at line 394, but I see it was just that this "i" was global that was raising the flag